.github/actions/build-sample-app/action.yml

```diff
@@ -100,7 +100,14 @@ runs:
       shell: bash
       run: cat $GITHUB_EVENT_PATH
 
-    - name: Build app via Fastlane 
+    - name: Update app version to uniquely identify build for easier finding later. 
```

Comment:
Because I broke up the `ios build` function into multiple functions to make it re-usable by SDK wrappers, we need to run the function `iossdk_update_app_version` before running `ios build` on the CI to make sample app builds.